src/js/pouch.js

```diff
@@ -0,0 +1,66 @@
+// Utility functions to add pouch to an existing express instance
+"use strict";
+var fluid = require("infusion");
+var gpii  = fluid.registerNamespace("gpii");
+fluid.registerNamespace("gpii.pouch");
+
+var PouchDB        = require("pouchdb");
+var memdown        = require("memdown");
+var expressPouchdb = require("express-pouchdb");
+
+gpii.pouch.init = function (that) {
+    var MemPouchDB = PouchDB.defaults({db: memdown });
+
+    if (that.options.databases && Object.keys(that.options.databases).length > 0) {
```

Comment:
A fluid.each(that.options.databases, function (dbConfig, key) {would allow you to dispense with this bulky if statement